client/src/components/SelectedIngredientsList/SelectedIngredientsList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import styles from "./styles.module.scss";
+import { FaRegCheckSquare } from "react-icons/fa";
+
+const SelectedIngredientsList = ({
+  selectedIngredients,
+  setSelectedIngredients,
+}) => {
+  const handleIngredientClick = (ingredientToRemove) => {
+    const updatedIngredients = selectedIngredients.filter(
+      (ing) => ing !== ingredientToRemove
+    );
+    setSelectedIngredients(updatedIngredients);
+  };
+  return (
+    // <div className={styles.wrapper}>
```

Comment:
It is important to delete unnecessary comments for the readability of the code.

Reply:
Good catch! Thank you Beyza!